.ci/Jenkinsfile-integration-test

```diff
@@ -0,0 +1,44 @@
+pipeline {
+    agent any
+
+    triggers {
+        cron(BRANCH_NAME == "master" ? "H H(2-5) * * *": "")
+    }
+
+    environment {
+        REPO_NAME="inmanta-dev-dependencies"
+    }
+
+    options {
+        disableConcurrentBuilds()
+        checkoutToSubdirectory(env.REPO_NAME)
+        skipDefaultCheckout()
+    }
+
+    stages {
+        stage("Setup") {
+            steps {
+                deleteDir()
+                dir(env.REPO_NAME) {
+                    checkout scm
+                    script {
+                        rm -rf ${WORKSPACE}/env
+                        python3 -m venv ${WORKSPACE}/env
+                        ${WORKSPACE}/env/bin/python3 -m pip install -U pip poetry
+                    }
+                }
+            }
+        }
+
+        stage("Poetry install") {
+            dir(env.REPO_NAME) {
+                script {
+                    sh '''
+                        poetry install --extras "async extension module"
+                        poetry lock
```

Comment:
Isn't this line redundant?

Reply:
Indeed it is, sorry for the misleading instructions in the original ticket